.releaserc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  branches: [
+    // https://github.com/semantic-release/semantic-release/blob/master/docs/usage/workflow-configuration.md#workflow-configuration
+    { name: 'master', channel: 'latest' },
+    { name: 'beta', channel: 'beta', prerelease: 'beta' }
+  ],
+  plugins: [
+    ['@semantic-release/commit-analyzer', {
+      releaseRules: [
+        { scope: 'no-release', release: false }
```

Comment:
Caso seja criado commit com tipo que gera nova versão, porém não há o desejo de publicar nova versão quando for mergeado com a master/beta, é possível utilizar o escopo `no-release`.

Exemplo de commit que sem o escopo geraria nova versão:

`fix(no-release): change logic xpto`

Tipos que geram nova versão:
- `feat`
- `fix`
- `performance`
- `revert`
- com texto `BREAKING CHANGE`